packages/chakra-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -73,7 +73,7 @@ export default function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFS
       })
     : [];
 
-  const isMultiple = typeof multiple !== 'undefined' && Boolean(enumOptions);
+  const isMultiple = typeof multiple !== 'undefined' && multiple !== false && Boolean(enumOptions);
```

Comment:
Please update the `CHANGELOG.md` file to add a line about this fix. Also, is there an existing bug that this is fixing? IF so, please reference it in the changelog as well as the PR description